substrate/frame/alliance/src/benchmarking.rs

```diff
@@ -19,21 +19,18 @@
 
 #![cfg(feature = "runtime-benchmarks")]
 
-use core::{cmp, mem::size_of};
-use sp_runtime::traits::{Bounded, Hash, StaticLookup};
-
-use frame_benchmarking::{account, v2::*, BenchmarkError};
-use frame_support::traits::{EnsureOrigin, Get, UnfilteredDispatchable};
-use frame_system::{pallet_prelude::BlockNumberFor, Pallet as System, RawOrigin as SystemOrigin};
-
 use super::{Call as AllianceCall, Pallet as Alliance, *};
+use core::{cmp, mem::size_of};
+use frame::{benchmarking::prelude::*, traits::UnfilteredDispatchable};
+use frame_system::Pallet as System;
```

Comment:
As in my previous observation, please check if `assert_last_event` is widely used and should be added to the benchmarking prelude.

Reply:
`frame::benchmarking::prelude::*;` imports the top level prelude, which exports the whole `frame_system` crate.

As investigations go;

- I see `assert_last_event` being used across `benchmarking.rs`, `tests.rs`, `lib.rs` files and many others. The commonality between these respective preludes is the Top level prelude which they all share.

- `assert_last_event` should be an export of the top level prelude, as it is a gated function https://github.com/paritytech/polkadot-sdk/blob/48c6574b1a32893e6a2113622d009cefda0a5f21/substrate/frame/system/src/lib.rs#L2016 but we should export it is `frame_system::Pallet as System` this gives use all secondary trait implementation and the base implementation. 

- This avoids overly adjusting pre-existing code.

wdyt @re-gius @kianenigma ?

Reply:
We could leave things as

```rust
use frame::benchmarking::prelude::{*, RawOrigin as SystemOrigin, frame_system::Pallet as System};
```

until we migrate most pallets?  @re-gius @kianenigma

Reply:
`assert_last_event` are sometimes instantiable, creating one function for both occurrences?

please could you explain a bit more